lib/get.js

```diff
@@ -35,7 +35,7 @@ module.exports = function get(options, callback) {
         }
         // request 2.2.x *always* returns the response body as a string.
         // @TODO remove this once request is upgraded.
```

Comment:
I believe this comment is now irrelevant 🕺

Reply:
This is outside the scope of this PR, but while we're on the topic of irrelevance, `maxKeepAliveRequests` as it's used here has not been a valid option to pass to `keepaliveagent` for quite some time:

https://github.com/mapbox/geojson-mapnikify/blob/dd20ead2d7f4da442cc5dfdfab12bec9f2a2c461/lib/get.js#L9

Reply:
nice flag-- we should update this as well!